src/pages/latest_tweets.tsx

```diff
@@ -0,0 +1,57 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import { Flex, Text } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { TweetData } from 'types/Tweet';
+import TweetInfo from 'components/TweetInfo';
+
+const LatestTweets: NextPage = () => {
+  const [data, setData] = useState<TweetData[]>([]);
+  const [isLoading, setLoading] = useState(false);
+
+  useEffect(() => {
```

Comment:
Why not serverSideProps?

Reply:
Im not very used to nextjs, I will replace to serverSideProps.